### PR TITLE
Update langchain

### LINKED ISSUE #89: Upgrade langchain
`pip install jupyter_ai`

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pyspark-ai 0.1.12 requires langchain<0.0.202,>=0.0.201, but you have langchain 0.0.220 which is incompatible.

### PR DESCRIPTION
**Pull request by bjornjorgensen**
fix #89

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ requests = "^2.31.0"
 tiktoken = "0.4.0"
 beautifulsoup4 = "^4.12.2"
 openai = "^0.27.8"
-langchain = "^0.0.201"
+langchain >= 0.0.220, < 0.1.0
```
Comment (gengliangwang) [inline pyproject.toml:30]:
```suggestion
langchain = ">=0.0.201,<0.1.0"
```

Comment (gengliangwang) [reply pyproject.toml:30]: @bjornjorgensen Thanks for the fix. It requires quoting here.

Comment (gengliangwang) [reply pyproject.toml:30]: ok, I am committing the suggestion now. Let me know if you are against it.

**Review by gengliangwang — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (gengliangwang) [reply pyproject.toml]: Yes, it works.

Comment (bjornjorgensen) [reply pyproject.toml]: Can this have anything to do with pyspark-ai https://github.com/langchain-ai/langchain#breaking-changes-for-select-chains-sqldatabase-on-728

Comment (gengliangwang) [reply pyproject.toml]: Not sure about that. Feel free to create a PR to increase the minimal required langchain version if you think it is necessary.

**Pull request merged**